File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task21_CLAUDE_claude-3-5-sonnet-20240620.java

import java.util.ArrayList;
import java.util.List;

public class Solution {
    public static List<Integer> removeSmallest(List<Integer> numbers) {
        if (numbers.isEmpty()) {
            return new ArrayList<>();
        }
        
        List<Integer> result = new ArrayList<>(numbers);
        int minIndex = 0;
        for (int i = 1; i < result.size(); i++) {
            if (result.get(i) < result.get(minIndex)) {
                minIndex = i;
            }
        }
        result.remove(minIndex);
        return result;
    }
}

File: dataset/output-bk/CLAUDE_claude-3-5-sonnet-20240620/Task21_CLAUDE_claude-3-5-sonnet-20240620.c

#include <stdlib.h>

int* removeSmallest(int* numbers, int numbersSize, int* resultSize) {
    if (numbersSize == 0) {
        *resultSize = 0;
        return NULL;
    }

    int* result = (int*)malloc((numbersSize - 1) * sizeof(int));
    int minIndex = 0;
    for (int i = 1; i < numbersSize; i++) {
        if (numbers[i] < numbers[minIndex]) {
            minIndex = i;
        }
    }

    int j = 0;
    for (int i = 0; i < numbersSize; i++) {
        if (i != minIndex) {
            result[j++] = numbers[i];
        }
    }

    *resultSize = numbersSize - 1;
    return result;
}
